### PR TITLE
IA REFRESH - (DOCSP-26922) [Port] Develop & Deploy Apps (/apps)

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-26922) [Port] Develop & Deploy Apps (/apps)

### Staged Changes

- [Develop & Deploy Apps](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps/apps/)

### CODE REVIEW

File: source/alerts.txt [hunk 1]
```diff
@@ -76,11 +76,11 @@ several **Events**:
          for Static Hosting
      - A Custom URL is confirmed for static hosting 
    
-   * - :ref:`Deployment <application-deployment>` 
+   * - :ref:`Deployment <deploy>`
```
Comment (mongodben) [inline source/alerts.txt:79]: is this ref deleted? 

if so, should we keep it and put it so there are no potential downstream breakages?

File: source/apps.txt [hunk 2]
```diff
@@ -20,15 +22,99 @@ Develop & Deploy Apps
...
+Overview
+--------
+
+What is an App Services App?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An App Services App is a collection of managed services that run your
+application's backend operations on MongoDB Atlas. You configure the
+App's services using declarative configuration files and write
+JavaScript code to implement custom behavior.
```
Comment (mongodben) [inline source/apps.txt:34]: nice description of App Services

File: source/apps.txt [hunk 3]
```diff
@@ -20,15 +22,99 @@ Develop & Deploy Apps
...
+
+An App sits between your application's frontend client and your data
+stored in MongoDB Atlas. Apps have built-in tools for modeling and
+working with data in Atlas, including role-based access permissions and
+document schema validation. You can use these to power services like
+Device Sync, Triggers, and the GraphQL and Data APIs.
```
Comment (mongodben) [inline source/apps.txt:40]: should we ref to these landing pages?

File: source/apps.txt [hunk 4]
```diff
@@ -20,15 +22,99 @@ Develop & Deploy Apps
...
+
+To learn how to create an App, see :ref:`Create an App <create-app>`.
+
+How are Apps Deployed?
+~~~~~~~~~~~~~~~~~~~~~~
+
+Apps use a "serverless" architecture, which means that you as the
+developer don't directly manage or think about the servers that run your
+App. Instead, you deploy your App to a one or more cloud regions around
+the world using an admin tool. Your services run on-demand on servers in
```
Comment (mongodben) [inline source/apps.txt:50]:
```suggestion
the world. Your services run on-demand on servers in
```

File: source/apps/rollback.txt [hunk 5]
```diff
@@ -1,4 +1,4 @@
-.. _roll-back-deployments:
+.. _rollback-deployments:
```
Comment (mongodben) [inline source/apps/rollback.txt:1]: maybe keep ref in case linked to from somewhere downstream

**Review by mongodben — APPROVED** [review]
lgtm. couple small things. nice work! love how this section is lookin

File: source/apps.txt [hunk 6]
```diff
@@ -20,15 +22,99 @@ Develop & Deploy Apps
    App Deployment Methods </apps/deploy>
    Deployment Models & Regions </apps/deployment-models-and-regions>
```
Comment (mongodben) [inline source/apps.txt:24]: can you add a right side TOC?

File: source/alerts.txt (hunk 1 above)
Comment (nlarew) [reply source/alerts.txt:79]: There are no downstream references in other docs repos and I'd prefer to not keep dangling refs around

File: source/apps/rollback.txt (hunk 5 above)
Comment (nlarew) [reply source/apps/rollback.txt:1]: I checked there's no downstream refs

**Pull request merged**